doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,272 @@
+
+```
+properties ::= properties_list
+properties_list ::= properties_entry ";" properties_list | properties_entry
+properties_entry ::= interface_name path
+```
+
+The following example is the inflated previous payload:
+
+```
+com.example.MyInterface/some/path;org.example.DraftInterface/otherPath
+```
+
+This protocol feature is fundamental when a device has any interface with an *allow_uset* mapping, *purge properties* allows to corrent any error due to unhandled unset messages.
+
+## Publishing Data
+
+Either Astarte or a device might publish new data on a interface/endpoint specific topic.
+The topic is built using `/<interface name>/<path>` schema, and it is used regardless the type of interface or mapping.
+
+Also `/` path is a valid path for object aggregated interfaces.
+
+Following topics are valid:
+
+* `com.example.MyInterface/some/path`
+* `org.example.DraftInterface/otherPath`
+* `com.example.astarte.ObjectAggregatedInterface/`
+
+Data messages QoS is chosen according to mapping settings, such as reliability. Properties are always published using QoS 2.
+
+| Interface Type | Reliability     | QoS |
+|----------------|-----------------|-----|
+| properties     | always unique   | 2   |
+| datastream     | unreliable      | 0   |
+| datastream     | guaranteed      | 1   |
+| datastream     | unique          | 2   |
+
+### Payload Format
+
+Payload format might change according to the message type. Payloads are always BSON encoded, except for unset messages that are empty.
+
+#### Property Message
+
+Property messages have a "v" key (which means value) and they look like the followings:
+
+* `{"v": "string property value"}`
+* `{"v": 10}`
+* `{"v": true}`
+
+Previous payloads are in order BSON encoded as following hex dumps:
+
+```
+0000000 22 00 00 00 02 76 00 16 00 00 00 73 74 72 69 6e
+0000020 67 20 70 72 6f 70 65 72 74 79 20 76 61 6c 75 65
+0000040 00 00
+```
+
+```
+0000000 0c 00 00 00 10 76 00 0a 00 00 00 00
+```
+
+```
+0000000 09 00 00 00 08 76 00 01 00
+```
+
+Property messages order must be preserved and they must be consumed in order, also devices must not publish data for server owned interfaces and viceversa.
+The same property with the same value can be sent several times, this behavior is allowed but discouraged: it's up to the device to avoid useless messages.
+A device must also make sure to publish all the properties that have been changed while the device was offline.
+
+#### Unset Property Message
+
+Properties can be unset with an unset message. An unset message is just an empty 0 bytes payload.
+
+#### Datastream Message (individual aggregation)
+
+Datastream messages for not object aggregated interfaces have a "v" key and an optional "t" key (which means timestamp) and they look like the followings:
+
+* `{"v": false}`
+
+* `{"v": 16.73}`
+
+* `{"v": 16.73, "t": "2018-09-20T13:17:02.890869Z"}`
+
+Timestamps are UTC timestamps (BSON 0x09 type), not ISO 8601 strings, when not provided reception timestamp is used.
```

Comment:
Instead of "not ISO 8601 strings.", I'd write the timestamp in the example as the usual 64 bit number, to avoid confusion.